mmv1/third_party/terraform/tests/resource_cloud_identity_group_membership_test.go.erb

```diff
@@ -11,6 +10,165 @@ import (
 )
 
+func TestAccCloudIdentityGroupMembership_update(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"org_domain":    getTestOrgDomainFromEnv(t),
+		"cust_id":       getTestCustIdFromEnv(t),
+		"identity_user": getTestIdentityUserFromEnv(t),
+		"random_suffix": randString(t, 10),
+	}
+
+	vcrTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckCloudIdentityGroupMembershipDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCloudIdentityGroupMembership_update1(context),
+			},
+			{
+				Config: testAccCloudIdentityGroupMembership_update2(context),
+			},
+			{
+				Config: testAccCloudIdentityGroupMembership_update1(context),
+			},
+		},
+	})
+}
+
+func testAccCloudIdentityGroupMembership_update1(context map[string]interface{}) string {
+	return Nprintf(`
+resource "google_cloud_identity_group" "group" {
+  display_name = "tf-test-my-identity-group%{random_suffix}"
+
+  parent = "customers/%{cust_id}"
+
+  group_key {
+    id = "tf-test-my-identity-group%{random_suffix}@%{org_domain}"
+  }
+
+  labels = {
+    "cloudidentity.googleapis.com/groups.discussion_forum" = ""
+  }
+}
+
+resource "google_cloud_identity_group_membership" "basic" {
+  group    = google_cloud_identity_group.group.id
+
+  preferred_member_key {
+    id = "%{identity_user}@%{org_domain}"
+  }
+
+  roles {
+    name = "MEMBER"
+  }
+
+}
+`, context)
+}
+
+func testAccCloudIdentityGroupMembership_update2(context map[string]interface{}) string {
+	return Nprintf(`
+resource "google_cloud_identity_group" "group" {
+  display_name = "tf-test-my-identity-group%{random_suffix}"
+
+  parent = "customers/%{cust_id}"
+
+  group_key {
+    id = "tf-test-my-identity-group%{random_suffix}@%{org_domain}"
+  }
+
+  labels = {
+    "cloudidentity.googleapis.com/groups.discussion_forum" = ""
+  }
+}
+
+resource "google_cloud_identity_group_membership" "basic" {
+  group    = google_cloud_identity_group.group.id
+
+  preferred_member_key {
+    id = "%{identity_user}@%{org_domain}"
+  }
+
+  roles {
+    name = "MEMBER"
+  }
+
+  roles {
+    name = "MANAGER"
+  }
+}
+`, context)
+}
+
+func TestAccCloudIdentityGroupMembership_import(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"org_domain":    getTestOrgDomainFromEnv(t),
+		"cust_id":       getTestCustIdFromEnv(t),
+		"identity_user": getTestIdentityUserFromEnv(t),
+		"random_suffix": randString(t, 10),
+	}
+
+	vcrTest(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		CheckDestroy: testAccCheckCloudIdentityGroupMembershipDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCloudIdentityGroupMembership_import(context),
+			},
+			{
+				ResourceName:            "google_cloud_identity_group_membership.basic",
+				ImportState:             true,
+				ImportStateVerify:       true,
+			},
+			{
```

Comment:
This config step is redundant I think, it should be done implicitly both by the config step & import step before this.

Reply:
Actually I don't know what exactly `ImportStateVerify` does. Is it same as "checking there's no plan diff"?

Reply:
✅ 669c31729867ca3610ba4b3c80687c678fe43236